streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBWindowStoreTest.java

```diff
@@ -87,6 +90,7 @@
 
     private final File baseDir = TestUtils.tempDirectory("test");
     private final MockProcessorContext context = new MockProcessorContext(baseDir, Serdes.ByteArray(), Serdes.ByteArray(), recordCollector, cache);
+    private WindowStore windowStore;
```

Comment:
Seems all test cases are creating the store with the same key-value type: `<Integer, String>`, the difference only lies in the constructor parameters. In this case could we still declare it as typed and hence remove the `@SuppressWarnings("unchecked")`?